<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAsFile"/>
    <addaction name="separator"/>
    <addaction name="pageSet"/>
    <addaction name="print"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pastAction"/>
    <addaction name="deleteAction"/>
    <addaction name="separator"/>
    <addaction name="findAction"/>
    <addaction name="nextFindAction"/>
    <addaction name="replaceAction"/>
    <addaction name="gotoAction"/>
    <addaction name="separator"/>
    <addaction name="allAction"/>
    <addaction name="dataAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="autoAction"/>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionS"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsFile">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="pageSet">
   <property name="text">
    <string>页面设置(&amp;U)</string>
   </property>
  </action>
  <action name="print">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="pastAction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除全部(&amp;L)</string>
   </property>
  </action>
  <action name="findAction">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="nextFindAction">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="replaceAction">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="gotoAction">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
  </action>
  <action name="allAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="dataAction">
   <property name="text">
    <string>日期(&amp;D)</string>
   </property>
  </action>
  <action name="autoAction">
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="actionS">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(&amp;M)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
